cli/rt/40_testing.js

```diff
@@ -97,6 +98,9 @@ finishing test case.`;
     };
 
     if (typeof t === "string") {
+      if (!options) {
+        throw new TypeError("Missing test options");
+      }
```

Comment:
All you're doing here is checking that a third argument is given, and it should be optional anyway.

Reply:
That's what I thought and I did it anyway... stupid.
Anyway I'm not sure if I need to do anything in this file, but I added a unitest at `cli/tests/unit/testing_test.ts`.